hack/make-rules/test-e2e-node.sh

```diff
@@ -154,10 +154,17 @@ if [ ${remote} = true ] ; then
   exit $?
 
 else
-  # Refresh sudo credentials for local run
-  if ! ping -c 1 -q metadata.google.internal &> /dev/null; then
-    echo "Updating sudo credentials"
-    sudo -v || exit 1
+  # Refresh sudo credentials if needed
+  if ping -c 1 -q metadata.google.internal &> /dev/null; then
+    echo 'Running on CGE, not asking for sudo credentials'
+  elif sudo --non-interactive "$(which /bin/bash)" -c true 2> /dev/null; then
```

Comment:
```suggestion
  elif sudo --non-interactive "$(which bash)" -c true 2> /dev/null; then
```

Reply:
Not sure why I did this in the 1st place... Good catch, thanks! :)